packages/riipen-ui/src/components/ButtonBase.jsx

```diff
@@ -0,0 +1,84 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+const ButtonBase = ({
+  ariaLabel,
+  classes,
+  children,
+  type,
+  variant,
+  ...other
+}) => {
+  const className = clsx("button", variant, classes);
+  return (
+    <>
+      <button
+        aria-label={ariaLabel}
+        className={className}
+        type={type}
+        {...other}
+      >
+        {children}
+      </button>
+      <style jsx>{`
+        .basic {
+        }
+
+        .button {
+          cursor: pointer;
+        }
+
+        .button:focus {
+          outline: 5px auto -webkit-focus-ring-color;
+        }
+
+        .default {
```

Comment:
as this is the intended/most likely to be used variant, I made default be the variant with cleared button styles